cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -268,3 +338,84 @@ func (o *CreateOptions) createObject(ctx context.Context, clientGetter util.Clie
 
 	return createdObj, nil
 }
+
+func (o *CreateOptions) RunInteractivePod(ctx context.Context, clientGetter util.ClientGetter, podName string) error {
+	k8sclient, err := clientGetter.K8sClientset()
+	if err != nil {
+		return err
+	}
+
+	if o.RemoveInteractivePod {
+		defer func() {
+			err = k8sclient.CoreV1().Pods(o.Namespace).Delete(ctx, podName, metav1.DeleteOptions{})
+			if err != nil {
+				fmt.Fprintln(o.ErrOut, err.Error())
+			}
+			fmt.Fprintf(o.Out, "pod \"%s\" deleted\n", podName)
+		}()
+	}
+
+	fmt.Fprintf(o.Out, "waiting for pod \"%s\" to be running...\n", podName)
+	err = waitForPodRunning(ctx, k8sclient, o.Namespace, podName, o.PodRunningTimeout)
+	if err != nil {
+		return err
+	}
+
+	pod, err := k8sclient.CoreV1().Pods(o.Namespace).Get(ctx, podName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	err = attachTTY(o, pod)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func attachTTY(o *CreateOptions, pod *corev1.Pod) error {
+	o.Stdin = true
+	o.TTY = true
+	containerToAttach := &pod.Spec.Containers[0]
+	if !containerToAttach.TTY {
```

Comment:
Not sure that we need to check that user set this parameter. It will obligate user to put this parameter manually every time. But we also know that it must be for interactive mode. Or maybe set this parameter automatically on builder, if it's required on attach. WDYT?

Reply:
Yeah, that flag should be enabled by the admin in the template. However, I agree, since we're in interactive mode, I guess we can set this param automatically regardless user config for TTY. I'd go for setting it on the builder.

Reply:
Done. Now TTY and STDIN are enabled in the builder for interactive mode.

Reply:
Thanks